api/src/main/java/org/pmiops/workbench/db/dao/DataSetServiceImpl.java

```diff
@@ -533,16 +544,10 @@ private String getQualifiedColumnName(Domain currentDomain, String columnName) {
     String prerequisites;
     switch (kernelTypeEnum) {
       case R:
-        prerequisites =
-            // RW-4241 workaround: update when the Jupyter image with the reticulate fix is fully
-            // rolled out
-            "require(devtools)\n"
-                + "devtools::install_github(\"rstudio/reticulate\", ref=\"00172079\")\n"
-                + "library(reticulate)\n"
-                + "pd <- reticulate::import(\"pandas\")";
+        prerequisites = "library(bigrquery)";
```

Comment:
Do we need to upgrade our notebook docker image to include bigrquery?

Reply:
I think the ticket said we don't have to in this case... Don't quote me on that.

Reply:
It functions without it, so we should be good. My guess is Leo includes it by default?

Reply:
Great.  I'll add it to my sanity-checking notebook.